### PR TITLE
Update Set-OwaMailboxPolicy.md

### PR DESCRIPTION
**Pull request by Benny-54**
This article is updated as per TASK 6527407, thank you. Please don't merge it yet, this needs to be reviewed by SME.

### CODE REVIEW

File: exchange/exchange-ps/exchange/Set-OwaMailboxPolicy.md [hunk 1]
```diff
@@ -1412,6 +1413,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OneWinNativeOutlookEnabled
+This parameter is available only in the cloud-based service.
+
+The OneWinNativeOutlookEnabled parameter controls the availability of the new Outlook for Windows app. Valid values are:
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-OwaMailboxPolicy.md:1419]:
```suggestion
The OneWinNativeOutlookEnabled parameter controls the availability of the new Outlook for Windows App. Valid values are:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Benny-54, kindly commit the suggested change. Many thanks.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrisda

**Comment by chrisda** [comment]
Has the SME reviewed this yet? I know about the new 'Monarch' Outlook app; it's likely more of a question of availability.

**Review by tseward — APPROVED** [review]

**Comment by Benny-54** [comment]
> Has the SME reviewed this yet? I know about the new 'Monarch' Outlook app; it's likely more of a question of availability.

Hi Chris,
Could you please revert this? this is not yet approved for publish by SME.

Thank you!

**Pull request merged**